src/services/navigationBar.ts

```diff
@@ -55,12 +58,12 @@ namespace ts.NavigationBar {
     const parentsStack: NavigationBarNode[] = [];
     let parent: NavigationBarNode;
 
-    function rootNavigationBarNode(sourceFile: SourceFile, cancellationToken: CancellationToken): NavigationBarNode {
+    function rootNavigationBarNode(sourceFile: SourceFile): NavigationBarNode {
         Debug.assert(!parentsStack.length);
         const root: NavigationBarNode = { node: sourceFile, additionalNodes: undefined, parent: undefined, children: undefined, indent: 0 };
         parent = root;
         for (const statement of sourceFile.statements) {
-            addChildrenRecursively(statement, cancellationToken);
+            addChildrenRecursively(statement, curCancellationToken);

```

Comment:
why do you need to pass it explicitly, it should be visible inside `addChildrenRecursively`